andela/vale-ah-backend
# #164139783 List User Functionalities

#### What does this PR do?
List User Functionalities
#### Description of Task to be completed?
- write unit test
- Create a route for listing all authors
- Create a route for viewing an author by username
- validate inputs with `joi`
#### How should this be manually tested?
- ensure that the build is passing on travis 
- Test endpoint api/profiles
- Test endpoint api/profiles/username
#### What are the relevant pivotal tracker stories?
[#164139783](https://www.pivotaltracker.com/story/show/164139783)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
